### PR TITLE
hwdef: disable GPS drivers on low flash boards

### PR DESCRIPTION
**Pull request by tridge**
only leave uBlox enabled on boards that are running out of flash

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# include file to reduce flash by including less GPS drivers
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc:1]:
```suggestion
# include file to reduce flash by including fewer GPS drivers
```

dammit

File: libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+define AP_GPS_BACKEND_DEFAULT_ENABLED 0
+define AP_GPS_UBLOX_ENABLED 1
+
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc:5]: Are you sure you don't want to include mavlink here?  It's damned useful for testing.

**Review by peterbarker — APPROVED** [review]
LGTM

File: libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc (hunk 2 above)
Comment (tridge) [reply libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc:5]: it is, but very few end users will ever use it, so I decided better to have more flash space

**Pull request merged**